flowtools/flowclr_sample_name/sample_simp.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="flclr_sample_name" name="Add sample information" version="1.0+galaxy1", profile="18.01">
+  <description>to flow clustered text file collection.</description>
+  <requirements>
+    <requirement type="package" version="1.42.0">bioconductor-flowcore</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="2:" level="fatal" description="See stderr for more details." />
+  </stdio>
+  <command><![CDATA[
+    Rscript '$__tool_directory__/sample_simp.R' '${input}' '${output}' '${samp_names}'  
+  ]]>
+  </command>
+  <inputs>      
+    <param format="flowclr" name="input" type="data_collection" collection_type="list" label="Clustered flowtext file collection"/>	
```

Comment:
Also, follow guidance on the [same link](https://planemo.readthedocs.io/en/latest/writing_advanced.html#processing-lists-reductions) which says that you should avoid setting the type to data_collection and instead use `multiple="true"`.